idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/service/PersistentSettings.kt

```diff
@@ -43,6 +43,7 @@ class PersistentSettings : SimplePersistentStateComponent<ValkyrieState>(Valkyri
         var addTrailingComma: Boolean by property(false)
 
         var showImageVectorPreview: Boolean by property(true)
+        var fileIndent: String? by string()
```

Comment:
Looks we can obtain an int by property here.

Reply:
However, the property does not seem to support the `Int?` type? If the user manually sets the indent size, there is no need to retrieve the indent size from the `.editorconfig` file.

Reply:
```kt
var fileIndent: Int by property(4)
```